.github/workflows/error-ref-publisher.yml

```
@@ -0,0 +1,65 @@
+name: Meshkit Error Codes Utility Runner
+on:
+  push:
+    branches:
+      - 'master'
+    paths:
+      - '**.go'
+
+jobs:
+  Update-error-codes:
+    name: Error codes utility
+    if: github.repository == 'layer5io/meshkit'
```

```suggestion
    if: github.repository == 'meshery/meshery-kuma'
```
this is the org/repo name for the repo in which the workflow will run.

---

I apologise for not noticing that.

---

No worries : )